src/cryptonote_core/service_node_list.h

```diff
@@ -74,9 +74,12 @@ namespace service_nodes
 
     struct service_node_info
     {
-      struct contribution {
+      struct contribution
+      {
         uint64_t amount;
-        cryptonote::account_public_address address;
+        uint64_t reserved;
+        uint8_t order;
+        contribution() : amount(0), reserved(0) { }
```

Comment:
Let's initialise `order` as well while we're here :)

Reply:
Sure, why not